docs/en/features/oauth.md

```diff
@@ -1,22 +1,29 @@
 # OAuth
 
-You can configure authentication services with external OAuth suppliers, right now Twitter, Facebook and Google are supported.
+You can configure authentication services with external OAuth providers, right now Twitter, Facebook, Google and Wordpress are supported.
 
 ## 1. Create an App on the platform
 
 For each platform, go to their developers section and follow their guides to create an app.
 
-## 2. Set your Consul Democracy's url
+## 2. Set your Consul Democracy's URL
```

Comment:
We've changed this text in Spanish but not in English :thinking:. Is it done on purpose?